src/Microsoft.AspNetCore.Mvc.Core/ResponseCacheAttribute.cs

```diff
@@ -78,6 +78,11 @@ public bool NoStore
         public string VaryByHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets the query keys used by the response cache middleware for creating secondary vary keys.
```

Comment:
I originally added the reference to the middleware to highlight that it's mandatory to add it. Otherwise, using this property will throw. But maybe we can just rely on the exception at runtime?